### DISCUSSION TITLE
RAPIDS configuration-Permission denied

### DISCUSSION THREAD

**Discussion by Evizaf** [carissalow/rapids#229] [Q&A]
Hi all,
I try to go through the configuration stages and I am stuck to : 'The script to connect and download data from this container '
I get this error:
src/data/streams/aware_mysql/container.R
/bin/sh: 5: src/data/streams/aware_mysql/container.R: Permission denied

any suggestions?
Thanks
Evi

**Comment by jenniferfedor** [comment]
Hi @Evizaf, thank you for using RAPIDS. The permission denied error suggests that you have to first make the script executable with `chmod +x`. However, running this container script is not necessary to configure RAPIDS. Can you please let us know which step of our [configuration](https://www.rapids.science/1.9/setup/configuration/) instructions you are on? Were you attempting to run this script independently for some reason or did this error occur during execution of the entire pipeline? Thanks!
